ref/general/health-check-microservices.adoc

```diff
@@ -11,11 +11,11 @@
 :seo-description:  A health check is a special REST API implementation that you can use to  validate the status of a microservice and its dependencies. MicroProfile Health enables services in an application to self-check their health and then publishes the overall health status to a defined endpoint.
 :page-layout: general-reference
 :page-type: general
-= Enabling health checking of microservices
+= Enabling microservice health checks
 
-Monitoring the health of the microservices that make up your application is crucial to providing your users with reliable functionality. A _health check_ is a special REST API implementation that you can use to validate the status of a microservice and its dependencies. link:https://github.com/eclipse/microprofile-health[MicroProfile Health] enables services in an application to self-check their health and then publishes the overall health status to a defined endpoint.
+Monitoring the health of the microservices that make up your application is crucial to providing your users with reliable performance. A _health check_ is a special REST API implementation that you can use to validate the status of a microservice and its dependencies. link:https://github.com/eclipse/microprofile-health[MicroProfile Health] enables services in an application to self-check their health and then publishes the overall health status to a defined endpoint.
 
-A self-check can assess anything that the service needs:
+A self-check can assess anything that the service needs, including the following factors:
```

Comment:
I disagree. Having a colon after "...needs:" signals the start of a list. "factors" doesn't mean anything and it's just adding padding to the sentence.